RetailCoder.VBE/UI/RubberduckUI.de.resx

```diff
@@ -1183,4 +1183,19 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren.</value>
   <data name="DisableThisInspection" xml:space="preserve">
     <value>Blockiere diese Untersuchung</value>
   </data>
+  <data name="ImplicitActiveWorkbookReference_" xml:space="preserve">
+    <value>'{0}' ist eine implizierte Referenz auf das 'ActiveWorkbook'</value>
+  </data>
+  <data name="ImplicitActiveSheetReference_" xml:space="preserve">
+    <value>'{0}' ist eine implizite Referenz auf das aktive Tabellenblatt</value>
```

Comment:
Scheint hier nicht konsistent mit Z. 1199
Vorschlag:

> '{0}' ist eine implizite Referenz auf das 'ActiveSheet'

---

Seems to be inconsistent against line 1199

Suggestion: (see above)